### PR TITLE
Introduce admin summary page

### PR DESCRIPTION
**Pull request by dnkrj**
⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

- Remove temporary design system pages
- Allow design system pages to specify breadcrumbs
- Add page located at `/admin/:document_type_slug` to act as a hub for changes to finders.
- Display guidance and a summary of the details of the specified finder.
- Test the contents of the summary displayed, including organisation lookup.

## Screenshot
specialist-publisher dev gov uk_admin_farming-grants(iPad Pro) (1)

Trello: https://trello.com/c/IGg2kTG5/3015-create-edit-finder-form-basic-details

### CODE REVIEW

File: spec/controllers/temp_controller_spec.rb [hunk 1]
```diff
@@ -1,16 +0,0 @@
-require "spec_helper"
```
Comment (ChrisBAshton) [inline spec/controllers/temp_controller_spec.rb:1]: I'd merge these first two commits.

File: app/views/admin/summary.erb [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+      items: [
+        {
+          label: "Finder documents",
+          href: documents_path(params[:document_type_slug]),
+        },
+        {
+          label: "Finder setup",
+          href: "/admin/#{current_format.admin_slug}",
+          current: true,
+        },
```
Comment (ChrisBAshton) [inline app/views/admin/summary.erb:38]: Nice work! Just to double-check, we're not intending to show this tabbed view on the 'main' finder page yet, right? (...until we pick up https://trello.com/c/JoK7tUbQ/443-release-the-edit-finder-forms-feature, so as not to point publishers to forms that aren't ready to use yet).

File: app/views/admin/summary.erb [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+
+    <%= render "govuk_publishing_components/components/inset_text", {
+    } do %>
+      <%= link_to("View on website (opens in new tab)", current_format.live_url, class: "govuk-link", target: "_blank", rel: "noopener") %>
+    <% end %>
+
+    <%= render "govuk_publishing_components/components/secondary_navigation", {
+      id: "nav_1234",
```
Comment (ChrisBAshton) [inline app/views/admin/summary.erb]: Looks like this ID can be dropped ([nowhere else in alphagov seems to specify it](https://github.com/search?q=org%3Aalphagov+components%2Fcomponents%2Fsecondary_navigation&type=Code).

PS the [page on publishing components](https://components.publishing.service.gov.uk/component-guide/secondary_navigation) could do with a more representative example, IMO - their tabs don't currently toggle any text 🤔

**Review by ChrisBAshton — APPROVED** [review]
LGTM - some nitpicks below. Thanks! ⭐

File: app/views/admin/summary.erb (hunk 3 above)
Comment (dnkrj) [reply app/views/admin/summary.erb]: Thanks!

File: app/views/admin/summary.erb (hunk 2 above)
Comment (dnkrj) [reply app/views/admin/summary.erb:38]: Yeah, no plans to add the tabs to the main finder page until this is ready for publishers to use

**Pull request merged**